src/storage/src/manifest/action.rs

```diff
@@ -38,8 +38,6 @@ pub struct RawRegionMetadata {
     pub columns: RawColumnsMetadata,
     pub column_families: RawColumnFamiliesMetadata,
     pub version: VersionNumber,
-    /// Time window for compaction
```

Comment:
Document the precision of this timestamp.

Reply:
> Document the precision of this timestamp.

This line was deleted